### ISSUE TITLE
Sth wrong when resume state wan 2.1 14B

### ISSUE THREAD

**Issue by lamphamnhd** [kohya-ss/musubi-tuner#307]
first of all, thanks kohya for your amazing work.

My problem: my PC got shutdown for some reason so I need to resume the lastest state to continue train on same epoch, same weight ect. but it seems like have sth really weirdo, even though my config already linked "--save_state --resume /workspace/wan/musubi-tuner/output/vanessaxoxo-000004-state", but as you see below, the model start at epoch 0 while i expected that it should start at 4. If anyone know  how i could get the model resume the lastest state correctly, please let me know! thanks!!!

My config:

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 wan_train_network.py --task t2v-14B --dit /workspace/wan/musubi-tuner/models/wan2.1_t2v_14B_bf16.safetensors --dataset_config /workspace/wan/musubi-tuner/dataset/dataset_config.toml --sdpa --fp8_base --fp8_scaled --fp8_t5 --mixed_precision bf16 --optimizer_type adamw8bit --learning_rate 2e-4 --gradient_checkpointing --max_data_loader_n_workers 1 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --network_alpha 32 --network_args "verbose=True" "loraplus_lr_ratio=4" --timestep_sampling shift --discrete_flow_shift 3.0 --max_train_epochs 20 --save_every_n_epochs 1 --seed 42 --blocks_to_swap 24 --output_dir /workspace/wan/musubi-tuner/output --output_name vanessaxoxo --vae /workspace/wan/musubi-tuner/models/wan_2.1_vae.safetensors --t5 /workspace/wan/musubi-tuner/models/models_t5_umt5-xxl-enc-bf16.pth --sample_prompts /workspace/wan/musubi-tuner/dataset/prompt.txt --sample_every_n_epochs 1 --log_config --log_with tensorboard --logging_dir logs --save_state --resume /workspace/wan/musubi-tuner/output/vanessaxoxo-000004-state

My log: 

INFO:accelerate.accelerator:Loading states from /workspace/wan/musubi-tuner/output/vanessaxoxo-000004-state
INFO:accelerate.checkpointing:All model weights loaded successfully
INFO:accelerate.checkpointing:All optimizer states loaded successfully
INFO:accelerate.checkpointing:All scheduler states loaded successfully
INFO:accelerate.checkpointing:All dataloader sampler states loaded successfully
INFO:accelerate.checkpointing:All random states loaded successfully
INFO:accelerate.accelerator:Loading in 0 custom states
running training / 学習開始
  num train items / 学習画像、動画数: 426
  num batches per epoch / 1epochのバッチ数: 426
  num epochs / epoch数: 20
  batch size per device / バッチサイズ: 1, 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 8520
INFO:hv_train_network:set DiT model name for metadata: /workspace/wan/musubi-tuner/models/wan2.1_t2v_14B_bf16.safetensors
INFO:hv_train_network:set VAE model name for metadata: /workspace/wan/musubi-tuner/models/wan_2.1_vae.safetensors
steps:   0%|                                                                                                         | 0/8520 [00:00<?, ?it/s]INFO:hv_train_network:DiT dtype: torch.bfloat16, device: cuda:0

epoch 1/20
INFO:dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1
INFO:dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1
steps:   0%|▎                                                                            | 28/8520 [05:12<26:20:47, 11.17s/it, avr_loss=0.112]

**Comment by gambikules** [comment]
not wron. That's how it works.

**Comment by Sarania** [comment]
Yeah this is correct behavior according to how accelerate manages epochs/steps when resuming. It shouldn't affect your results or anything you just have to mentally keep track at that point.
